src/test/phases/frenzy-move-reset.test.ts

```diff
@@ -0,0 +1,71 @@
+import { Abilities } from "#app/enums/abilities";
+import { BattlerTagType } from "#app/enums/battler-tag-type";
+import { StatusEffect } from "#app/enums/status-effect";
```

Comment:
```suggestion
import { BattlerIndex } from "#app/battle";
import { Abilities } from "#enums/abilities";
import { BattlerTagType } from "#enums/battler-tag-type";
import { StatusEffect } from "#enums/status-effect";
```
Whoops, forgot to add the import since this isn't VSCode lol.